Sources/Datadog/Logging/Log/LogEventSanitizer.swift

```diff
@@ -13,13 +13,8 @@ internal struct LogEventSanitizer {
         /// If any of those is used by the user, the attribute will be ignored.
         static let reservedAttributeNames: Set<String> = [
             "host", "message", "status", "service", "source", "ddtags",
-            TracingWithLoggingIntegration.TracingAttributes.traceID,
-            TracingWithLoggingIntegration.TracingAttributes.spanID,
-            RUMContextIntegration.Attributes.applicationID,
-            RUMContextIntegration.Attributes.sessionID,
-            RUMContextIntegration.Attributes.viewID,
-            LoggingWithActiveSpanIntegration.Attributes.traceID,
-            LoggingWithActiveSpanIntegration.Attributes.spanID,
+            "dd.trace_id", "dd.span_id",
+            "application_id", "session_id", "view.id", "user_action.id",
```

Comment:
I'm a bit afraid of switching from compiled constants to string literals. This way we lose compiler check on these values and it is easy to get out of sync and cause problems 🐞 (e.g. changing their value in definition, but not here). Also, we're losing self-documenting aspect of referencing constants in other types.

On the other hand, I understand that in modular setup, such references aren't possible. Do you have any thoughts on this @maxep ? By design, such things should be tested as integration between features, however this is a good example where such tests will be very hard to establish.

Reply:
Yes, I tend to remove any dependency between modules. But you are right, any change in the emitted context won't be catch by tests, the reserved attributes sanitisation is not currently tested.

But we can still test this as an integration test and check against constant defined in the emitting Feature, let me draft something!

Reply:
@ncreated I've covered this in tests, by creating constants in each Features and then test sanitisation of logs against those constants. It will catch any change made on the attributes.